### ISSUE TITLE
materialsCitation holotype not found zootaxa.4641.1.1

### ISSUE THREAD

**Issue by myrmoteras** [gsautter/goldengate-imagine#707]
in this example, the holotype has been missed

![image](https://user-images.githubusercontent.com/4609956/61527902-cd7ff780-aa1d-11e9-9461-259a8eb92006.png)

**Comment by myrmoteras** [comment]
but here in the same pub it is 

![image](https://user-images.githubusercontent.com/4609956/61528135-526b1100-aa1e-11e9-8a36-41ee3e0f6b75.png)

**Comment by gsautter** [comment]
This looks like a morphological problem to me: The problem with "Male holotype" is that as opposed to "Paratypes", "holotype" is not capitalized ... to facilitate grouping by and scoping of type status, there is a respective morphological filter in place, intended to get "holotype" as in "same data as holotype" out of the way. In the second example "Holotype male" the type status is capitalized and thus used.

Maybe we have to add a "specimenSex+typeStatus" rule to catch this ... However, for the articles I've seen, "Holotype male" is way more frequent than "Male holotype". Or am I mistaking?

**Comment by myrmoteras** [comment]
yes, we getting into the rare case section. in almost all cases it is Holotype
